2023/Q2/artifacts/PoS-quint/namada.qnt

```diff
@@ -493,7 +493,7 @@ module namada {
         pos' = result.posState,
         delegators' = delegators,
         lastTx' = {tag: "Evidence", result: true, user: "", validator: validator, amount: e},
-        nextInfractionEpoch' = nextInfractionEpoch.set(validator, e + UNBONDING_OFFSET)
+        nextInfractionEpoch' = nextInfractionEpoch.set(validator, e + UNBONDING_OFFSET) // should this have CUBIC_OFFSET?
```

Comment:
I think so to correspond to the offset of `enqueuedSlashes`

Reply:
This is only to control the pace at which infractions of a given validator can be submitted without preventing repeated infractions: a validator misbehaving more than once with the same stake. To prevent repeated infractions, we can set `limitEvidence=2`